src/HubSpot.Client/Authentication/ITokenSelector.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HubSpot.Authentication
+{
+    public interface ITokenSelector
+    {
+        string SelectToken(Dictionary<string, string> tokens);
+        bool IsTokenChanged();
+    }
+}
```

Comment:
Is the implementation for this in the Hubspot RA? (Cause I'm not seeing it here)

Reply:
Yes, exactly, reasoning being that each user of the sdk will probably have its own case of switching between tokens, also it was being difficult to make this dynamic since the inputs are coming from the RA appsettings.